ci_framework/roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -65,3 +65,29 @@ cifmw_cephadm_extra_args: ""
 cifmw_cephadm_pacific_filter: "16.*"
 cifmw_cephadm_dashboard_enabled: false
 cifmw_cephadm_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+# The path of the rendered rgw spec file
+cifmw_ceph_rgw_spec_path: /tmp/ceph_rgw.yml
+cifmw_ceph_rgw_network: 192.168.122.0/24
+cifmw_cephadm_rgw_vip: 192.168.122.22/24
```

Comment:
Can these lines be deleted?
```
cifmw_ceph_rgw_network: 192.168.122.0/24
cifmw_cephadm_rgw_vip: 192.168.122.22/24
```

Reply:
Yes, not required I guess: the first one is computed from `public_network` and the second is retrieved filtering the available IP address w/ the list of the already assigned IP addresses.

Reply:
Done